spec/dynamoid/persistence_spec.rb

```diff
@@ -739,13 +738,13 @@ def reload_address
   context 'delete' do
     it 'uses dumped value of sort key to call DeleteItem' do
       klass = new_class do
-        range :activated_at, :datetime
+        range :activated_on, :date
       end
 
-      obj = klass.create!(activated_at: Time.now)
+      obj = klass.create!(activated_on: Date.today)
```

Comment:
`Date.today` is dangerous and should generally be avoided as it is not Rails time zone aware.  Especially in specs where we might have people copying the patterns in their own code.  Use `Time.zone.today` instead.

Ref: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/Date